app/ui.js

```diff
@@ -8,15 +8,15 @@
  * See README.md for usage and integration instructions.
  */
 
-import * as Log from '../core/util/logging.js';
-import _, { l10n } from './localization.js';
-import { isTouchDevice } from '../core/util/browser.js';
-import { setCapture, getPointerEvent } from '../core/util/events.js';
-import KeyTable from '../core/input/keysym.js';
-import keysyms from '../core/input/keysymdef.js';
-import Keyboard from '../core/input/keyboard.js';
-import RFB from '../core/rfb.js';
-import * as WebUtil from './webutil.js';
+import * as Log from '../core/util/logging';
```

Comment:
These need to stay in.  Support for bare names is unclear in the WHATWG spec, and the module loader polyfill doesn't permit it: https://github.com/ModuleLoader/browser-es-module-loader

Reply:
I can bring it back.

It's weird though, that it hadn't fail any of the tests...